docs/API_DESIGN.md

```diff
@@ -530,6 +548,20 @@ uri = "did:dht:i9xkp8ddcbcg8jwq54ox699wuzxyifsqx4jru45zodqu453ksz6y"
 resolution_result = DidDht.resolve(uri)
 ```
 
+## `BearerDid`
+
+```pseudocode!
+CLASS BearerDid
+  PUBLIC DATA did: Did
+  PUBLIC DATA document: Document
+  CONSTRUCTOR(did: Did, key_manager: KeyManager)
```

Comment:
```suggestion
  CONSTRUCTOR(uri: string, key_manager: KeyManager)
```

For consistency, for now. IMO we should migrate every instance of `uri: string` to `did: Did`, that way the errors are pushed closer to the developer in the case the DID URI is invalid